fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -701,6 +701,13 @@ class ShardManager {
 					if (!shard->initialized()) {
 						continue;
 					}
+					uint64_t liveDataSize = 0;
+					ASSERT(shard->db->GetIntProperty(
+					    shard->cf, rocksdb::DB::Properties::kEstimateLiveDataSize, &liveDataSize));
+					TraceEvent(SevInfo, "PhysicalShardCFSize")
```

Comment:
Just curios how many shards we will have in prod? If it is some xxx times, this statement will print so many times.. Can we not club all shards in trace event?
Just a thought

Reply:
If there are too many trace events in the loop, they can be throttled to protect the server from OOM.